tests/components/number/test_init.py

```diff
@@ -686,6 +687,22 @@ async def test_restore_number_restore_state(
             100,
             38.0,
         ),
+        (
+            SensorDeviceClass.VOLUME_FLOW_RATE,
```

Comment:
@elupus Please open a follow-up PR and change this to `NumberDeviceClass.VOLUME_FLOW_RATE`

Reply:
https://github.com/home-assistant/core/pull/109501